### ISSUE TITLE
Suggestion: Monster types

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#93]
```
Having monster types will allow to specify properties that applies to all 
monsters of that type. This will later allow us to expand beyond plain physical 
damage (eg. poison), resistances (eg. ice spells) or other +/- abilities, which 
could be both for the monster or against the player (eg +1 DR or -1 AP 
(lowering monster AP from default 10 to 9)). It could also be a graphical 
thing, like they have glow effect.

Monster type would be included as a new line in the monster info box.

Examples
Snake
 |--> Venomous [snake]
 |--> Fire [snake]
Wasp
Undead
 |--> Skeleton
 |--> Zombie
Demon
 |--> Dragon
```

Original issue reported on code.google.com by `kims...@gmail.com` on 19 Dec 2010 at 10:32

**Comment by GoogleCodeExporter** [comment]
Original comment by `sdeva...@gmail.com` on 10 Feb 2011 at 1:29
- Added labels: **Type-Enhancement**, **Priority-Low**
- Removed labels: **Type-Defect**, **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Monster types are introduced in v0.6.11 . Initially used for immunity to 
critical hits. Might be used for specific skills in the future.

Marking this as ready.
```

Original comment by `oskar.wi...@gmail.com` on 29 Apr 2012 at 3:53
- Changed state: **Ready**
- Added labels: **Milestone-0.6.11**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
